_posts/2024-07-17-Beethoven-Tutorials.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "New Tutorials released for the Beethoven Annotator App"
+authors: "Mark Saccomano"
+excerpt: "Try out the new step-by-step guide to creating digital annotations on digital resources using MEI and Linked Data."
+teaserImage: "/assets/blog/2024-07-17/teaser.jpg"
+---
+
+The online Annotator App developed for the ViFE project [Beethoven in the House](https://domestic-beethoven.eu/) now has [documentation and tutorials available](https://domestic-beethoven.eu/tutorials/), thanks to funding obtained from NFDI4Culture. Beethoven in the House was a three-year collaborative research project between Edirom, the Oxford e-Research Centre, and the Beethoven Archive Research Center in Bonn. The project website now features tutorials that illustrate the use of the app, broken down into 5 short steps. 
```

Comment:
I'd suggest to replace "Edirom" with "Paderborn University" here:
```suggestion
The online Annotator App developed for the ViFE project [Beethoven in the House](https://domestic-beethoven.eu/) now has [documentation and tutorials available](https://domestic-beethoven.eu/tutorials/), thanks to funding obtained from NFDI4Culture. Beethoven in the House was a three-year collaborative research project between Paderborn University, the Oxford e-Research Centre, and the Beethoven Archive Research Center in Bonn. The project website now features tutorials that illustrate the use of the app, broken down into 5 short steps. 
```

Reply:
Looks good. I meant to ask about that...